src/browser/BrowserInfo.ts

```diff
@@ -29,6 +34,15 @@ export class BrowserInfo {
 		this.userAgent = userAgent;
 		this.zoneOffset = -new Date().getTimezoneOffset() * 60000;
 		this.hostName = window.location.hostname;
+		this.latestReferrer = window.document.referrer;
+		if (this.latestReferrer && this.latestReferrer !== '') {
+			try {
+				const url = new URL(this.latestReferrer);
+				this.latestReferrerHost = url.host;
+			} catch (error) {
+				logger.debug('parse latest referrer domain failed: ' + error);
```

Comment:
The log level should be `warn` or `error`

Reply:
changed to warn.